priv/repo/migrations/20230227183202_add_anonymous_field_to_posts_table.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Oli.Repo.Migrations.AddAnonymousFieldToPostsTable do
+  use Ecto.Migration
+
+  def change do
+    alter table(:posts) do
+      add :anonymous, :boolean, default: false
+    end
```

Comment:
Aren't we missing the migration for the anonymous_posting field in `collab_space_config`?

Reply:
There is no need to create a migration to add that field since `anonymous_posting` is a field of the map `collab_space_config` that is stored in the `delivery_settings` table (see the[ migration for that table](https://github.com/wyeworks/oli-torus/blob/feature/MER-1694-anonymous-posts-for-students/priv/repo/migrations/20221103162115_add_delivery_settings.exs)).